distribution/docs/src/main/jdocs/content/_plugins/metacard-groomer-contents.adoc

```diff
@@ -0,0 +1,31 @@
+:type: plugin
+:status: published
+:title: Metacard Groomer
+:link: _metacard_groomer
+:plugintypes: preingest
+:summary: Modifies metacards when created or updated.
+
+The Metacard Groomer Pre-Ingest plugin makes modifications to `CreateRequest` and `UpdateRequest` metacards.
+
+Use this pre-ingest plugin as a convenience to apply basic rules for your metacards. 
```

Comment:
I've started adding a relationship section. Might be worth listing some of the most important things that rely on this, so we can make it explicit.